Lib/test/test_getpass.py

```diff
@@ -201,5 +201,38 @@ def test_control_chars_with_echo_char(self):
         self.assertEqual('Password: *******\x08 \x08', mock_output.getvalue())
 
 
+class GetpassEchoCharTest(unittest.TestCase):
+    # Tests for the `echo_char` parameter of `getpass.getpass()`
+    # Successful Validation Case(s)
+    def test_accepts_none(self):
+        getpass._check_echo_char(None)
```

Comment:
```suggestion
class GetpassEchoCharTest(unittest.TestCase):

    def test_accepts_none(self):
        getpass._check_echo_char(None)
```

The name of the tests are sufficiently clear to know whether we're talking about succeess or expecting failures.